config/default.ts

```diff
@@ -36,4 +37,17 @@ export default {
     pageSizeDefault: 5,
     sortingDirectionDefault: 'ASC',
   },
+  mailService: {
```

Comment:
This settings should be moved to production. But frankly, I'd delete them. Dima is not in the project anymore. We don't even know if they're working.

By default, the Dummy service should be initialized.